src/main.rs

```diff
@@ -128,11 +133,22 @@ fn run() -> Result<()> {
 		return Ok(());
 	}
 
+	let log_level = match matches.opt_str("l").as_ref().map(String::as_ref) {
+		Some("0") => LogLevelFilter::Off,
+		Some("1") => LogLevelFilter::Error,
+		Some("2") => LogLevelFilter::Info,
+		Some("3") => LogLevelFilter::Debug,
+		_ => LogLevelFilter::Info,
+	};
+
+	TermLogger::init(log_level, Config::default()).unwrap();
```

Comment:
Can you output an error (manually or using `?`, whichever is most convenient) instead of unwrapping?